### ISSUE TITLE
remove fpm

### ISSUE THREAD

**Issue by robnagler** [biviosoftware/container-perl#3]
```txt
myum install ruby-devel bind
^[[0mSuccessfully installed cabin-0.9.0
Successfully installed backports-3.20.2
Successfully installed arr-pm-0.0.10
Successfully installed clamp-1.0.1
Building native extensions.  This could take a while...
^[[91mERROR:  Error installing fpm:
        git requires Ruby version >= 2.3.
```

**Comment by robnagler** [comment]
I need to handle this now...

```txt
++ bivio project link_facade_files
2021/02/27 02:04:12 JSON::eval2955:2 Bivio::DieCode::DIE: Can't locate JSON/XS.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/\
share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .)
die at (eval 2955) line 2.
        JSON::BEGIN() called at (eval 2955) line 2
        eval {...} called at (eval 2955) line 2
        eval '
        use JSON::XS 2.34 ();

;' called at /usr/share/perl5/vendor_perl/JSON.pm line 252
        JSON::_load_xs(1) called at /usr/share/perl5/vendor_perl/JSON.pm line 58
        require JSON.pm called at /usr/share/perl5/vendor_perl/Bivio/HTML/Scraper.pm line 9
        Bivio::HTML::Scraper::BEGIN() called at (eval 2955) line 2
        eval {...} called at (eval 2955) line 2
        require Bivio/HTML/Scraper.pm called at (eval 2946) line 4
        Bivio::IO::ClassLoader::__ANON__() called at (eval 2947) line 1
        eval 'package Bivio::IO::ClassLoader; $code->();' called at /usr/share/perl5/vendor_perl/Bivio/Die.pm line 454
        Bivio::Die::_eval('SCALAR(0x27aa508)') called at /usr/share/perl5/vendor_perl/Bivio/Die.pm line 118
```

**Comment by robnagler** [comment]
Fix be85bf79b04d37e54b173dae2d5fc2a742fcf2a9 but not completely with rpm-perl changes
